invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/DocumentDetails/components/DocumentAccessRestrictions/state/actions.js

```diff
@@ -0,0 +1,64 @@
+import { IS_LOADING, SUCCESS, HAS_ERROR } from './types';
+import { document as documentApi, patron } from '../../../../../../common/api';
+import {
+  sendErrorNotification,
+  sendSuccessNotification,
+} from '../../../../../../common/components/Notifications';
+import { fetchDocumentDetails } from '../../../state/actions';
+import { ES_DELAY } from '../../../../../../common/config';
+import isEmpty from 'lodash/isEmpty';
+
+export const searchPatrons = queryText => {
```

Comment:
I would say not to mix the concepts and move this in `PatronsSearch` but then there you would end up with a state folder that the `PatronsSearch` component doesn't use :/ But it feels a bit strange to be here...@ntarocco thoughts?